UI: Add --fullscreen-preview flag for startup

karlsoderback opened the pull request.

pkviet commented:
hi, 
something went wrong with your commits: the diffs are too large , as if you had removed the original files and replaced with other ones.

karlsoderback commented:
Hi,
Sorry didn't see that, I'll fix it. Should I remove this pull request for
now and make another when I have fixed it?

2017-12-29 14:11 GMT+01:00 pkviet <notifications@github.com>:

> hi,
> something went wrong with your commits: the diffs are too large , as if
> you had removed the original files and replaced with other ones.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jp9000/obs-studio/pull/1125#issuecomment-354444175>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AhKQiZ_I1zPrmazRDWonPCS5-tA8sVfeks5tFOTlgaJpZM4RO8dl>
> .
>

reboot commented:
The PR will be updated with whatever you push on the branch you used to create it. If you create a feature branch you can just force push the new changes. In your case you used your master branch, so it may be a problem. But you can still push only the change you want to it, by remvoing all other chages from your master branch. You can just not use the master branch for something else until the PR got merged. And in the future better use one branch per feature, so you won't get problems when you need to update them.
